app/src/main/java/co/copperhead/pdfviewer/PdfViewer.java

```diff
@@ -66,6 +88,125 @@ public int getZoomLevel() {
         public void setNumPages(int numPages) {
             mNumPages = numPages;
         }
+
+        @JavascriptInterface
+        public void setDocumentProperties(final String properties) {
+            new AsyncTask<Void, Void, Void>() {
+                private String formatProperty(String property, String propertyValue) {
+                    return String.format("\n%s:\n%s\n", property, propertyValue);
+                }
+
+                private String parseFileSize(long fileSize) {
+                    final double kb = fileSize / 1024;
```

Comment:
I think this should use 1000, for consistency with the units elsewhere in Android's storage / file interfaces. Using 1024 isn't wrong but then the kB / MB below should be kiB / MiB per the SI definitions (despite the historical computer usage, which isn't standard).